parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -67,6 +67,13 @@ fi
 
 configureAdminUser
 
+{{- if NeedsContainerd}}
+# If crictl gets installed then use it as the cri cli instead of ctr
+# crictl is not a critical component so continue with boostrapping if the install fails
+# CLI_TOOL is by default set to "ctr"
+installCrictl && CLI_TOOL="crictl"
+{{- end}}
+
```

Comment:
does this work for Mariner? maybe exclude that?

Reply:
also if crictl is not installed, it means it's raw ubuntu, do we still need that?

Reply:
There's never an absolute need for crictl. If the install fails bootstrapping will continue using ctr. The only part that actuall uses ctr/crictl is to clean up the cached images. 
For for raw ubuntu it'll just use ctr b/c installCrictl will fail.

Reply:
ok. if there is no image then no need to install crictl. 
I'm thinking about an implicit rule that we install nothing in CSE part to avoid potential egress break and to boost perf. wdyt?  If we do that, we can move the downloadcrictl to tools_install.sh and separate offline and runtime logic further.

Reply:
ok I think downloadCrictl is not beign called by CSE, let's move it to tools_install script in next PR